instances/widgets.treasury-factory.near/widget/pages/asset-exchange/CreateRequest.jsx

```diff
@@ -142,6 +139,27 @@ return (
       showInProgress={isTxnCreated}
       cancelTxn={() => setTxnCreated(false)}
     />
+
+    {/* Treasury Wallet Dropdown - Only show if showNearIntents is enabled */}
+    {showNearIntents && (
+      <div className="mb-3">
+        <label className="form-label">Treasury Wallet</label>
+        <Widget
+          src="${REPL_BASE_DEPLOYMENT_ACCOUNT}/widget/components.DropDownWithSearchAndManualRequest"
+          props={{
+            selectedValue: treasuryWallet,
```

Comment:
after my PR is merged I guess we can use the `WalletDropdown` component, or I will update it since your PR will be merged before

Reply:
yes I hope to merge my PR first, to be able to start the testing in staging